osu.Game.Rulesets.Osu/UI/OsuPlayfieldAdjustmentContainer.cs

```diff
@@ -15,6 +15,14 @@ public class OsuPlayfieldAdjustmentContainer : PlayfieldAdjustmentContainer
 
         private const float playfield_size_adjust = 0.8f;
 
+        /// <summary>
+        /// Whether an osu-stable playfield offset should be applied (8 osu!pixels)
```

Comment:
This should mention the reasoning for applying the offset (ie. for storyboard alignment).

Also we have not used the term "osu!pixels" anywhere else so please don't introduce it here. The value doesn't even need to be mentioned in the xmldoc because the description of what it does should be enough to satisfy someone setting this externally.

Reply:
So reword it to something like this?
```
Whether an osu-stable playfield offset should be applied.
Set to true if correct alignment with storyboards is needed.
```

Reply:
I'd rename the property to `AlignWithStoryboard`.

"When true, an offset is applied to allow alignment with historical storyboards displayed in the same parent space. This will shift the playfield downwards slightly."

Reply:
Yeah, that sounds much better